File: ejercicios parte 1/2)estructuras condicionales/12anios_bisiestos/anios_bisiestos.java
import java.util.Scanner;

public class anios_bisiestos {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int anio;
            System.out.print("Ingrese el año: ");
            anio = scan.nextInt();
            if(anio%4==0){
                System.out.println("Es un año bisiesto");
            }else{
                System.out.println("No es un año bisiesto");
            }
        }
    }    
}

File: ejercicios parte 1/2)estructuras condicionales/12anios_bisiestos/anios_bisiestos.cpp
#include <iostream>
using namespace std;
int main(){
    int anio; bool verificar=false;;
    cout<<"Ingrese el año: ";
    cin>>anio;
    if(anio%4==0)
        verificar=true;
    if(anio>=1582){
        if(anio % 100 == 0){
            verificar=false;
            if(anio % 400 == 0){
                verificar = true;
            }
        }

    }
    
    if(verificar){
        cout<<"El año es bisiesto"<<endl;
    }else{
        cout<<"El año NO es bisiesto"<<endl;
    }
    return 0;
}